src/tests/integration/test_generic_db_operations.py

```diff
@@ -320,6 +334,18 @@ def test_telemetry_execute_command_has_connection_info(
             "ip_with_MSSQL",
             marks=pytest.mark.xfail(reason="sqlglot does not support SQL server"),
         ),
+        pytest.param(
+            "ip_with_Snowflake",
+            marks=pytest.mark.xfail(
+                reason="Something wrong with sqlplot histogram in snowflake"
+            ),
```

Comment:
why is this test affecting snowflake? I think this was fixed already in a PR

Reply:
It's showing deleted in this commit since I had added by mistake. `ip_with_Snowflake` is already present in the `test_sqlplot_histogram`. You meant this test right? 

https://github.com/ploomber/jupysql/blob/adc32fe25cef3205836f6d3e789823931f117120/src/tests/integration/test_generic_db_operations.py#L330

From the commit:
<img width="837" alt="Screenshot 2023-08-23 at 3 53 52 PM" src="https://github.com/ploomber/jupysql/assets/29096135/2dd940ad-5de0-4e7d-b8ff-fc11a4925231">